src/components/common/SearchBox/SearchBox.tsx

```diff
@@ -41,43 +44,52 @@ export const SearchBox: React.FC<SearchBoxProps> = ({
   };
 
   return (
-    <SearchInput
-      size="small"
-      variant="outlined"
-      sx={{ width: { sm: '27ch', md: '31ch' } }}
-      onChange={(e) => handleOnChange(e.target.value)}
-      placeholder={placeholder ?? t('Search')}
-      value={currentSearchTerm}
-      InputProps={{
-        startAdornment: (
-          <InputAdornment position="start">
-            {showContactSearchIcon ? (
-              <Icon path={mdiAccountSearch} size={1} />
-            ) : (
-              <SearchIcon />
-            )}
-          </InputAdornment>
-        ),
-        endAdornment: (
-          <InputAdornment position="end">
-            {currentSearchTerm && (
-              <IconButton
-                onClick={() => handleOnChange('')}
-                data-testid="SearchInputCloseButton"
-              >
-                <Close
-                  titleAccess={t('Close')}
-                  sx={{
-                    width: 14,
-                    height: 14,
-                    color: 'text.primary',
-                  }}
-                />
-              </IconButton>
-            )}
-          </InputAdornment>
-        ),
-      }}
-    />
+    <Tooltip
+      arrow
+      title={
+        page === PageEnum.Task
+          ? t('Search subject, tags, contact name, comments')
+          : t('Search name, phone, email, partner #')
```

Comment:
What do you think about this tweak?

```suggestion
          ? t('Search by subject, tags, contact name, or comments')
          : t('Search by name, phone, email, or partner #')
```